### DISCUSSION TITLE
File.drawio is not loading in the mkdocs material

### DISCUSSION THREAD

**Discussion by Sahayavinodh** [squidfunk/mkdocs-material#7245] [Q&A]
I have File.drawio in the mkdocs, i added the necessary plugins but the file is not loading on the website. When I refresh the webpage, the file gets loaded. Can you please provide the solution for this?

**Comment by alexvoss** [comment]
Without something to go on we cannot help. If you can create a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) then we can look at that. Note, though, that this may not be a problem with Material (which you could test by switching to another theme), so you may need to take it to the authors of the plugins you are using.

**Comment by copdips** [comment]
If you use vscode, you can install the extension [hediet.vscode-drawio](https://marketplace.visualstudio.com/items?itemName=hediet.vscode-drawio),
that supports `filename.drawio.png` format, which is a valid PNG image, but the extension can edit it directly.
